common/network-shuffle/src/test/java/org/apache/spark/network/shuffle/RetryingBlockTransferorSuite.java

```diff
@@ -245,8 +269,9 @@ private static void performInteractions(List<? extends Map<String, Object>> inte
     throws IOException, InterruptedException {
 
     MapConfigProvider provider = new MapConfigProvider(ImmutableMap.of(
-      "spark.shuffle.io.maxRetries", "2",
-      "spark.shuffle.io.retryWait", "0"));
+        "spark.shuffle.io.maxRetries", "2",
+        "spark.shuffle.io.retryWait", "0",
+        "spark.shuffle.sasl.enableRetries", "true"));
```

Comment:
nit: fix indentation.

Also, we want to make sure tests are handling `spark.shuffle.sasl.enableRetries = false` case correctly as well.
You can extend the test and specify value of enableRetries = true there

See `LevelDBHybridStoreSuite` as an example

Reply:
I don't think we can follow the same pattern here as `LevelDBHybridStoreSuite` because the test logic between the 2 cases is also different. It would make sense to extend if the test logic didn't differ. I ended up making the config map a private static field that can be modified at a per test level. This should be extensible to future tests that are added that need to add new configs as well.